pkgs/applications/science/math/R/default.nix

```diff
@@ -49,10 +53,22 @@ stdenv.mkDerivation rec {
       LDFLAGS="-L${gfortran.cc}/lib"
       RANLIB=$(type -p ranlib)
       R_SHELL="${stdenv.shell}"
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+      --without-tcltk
+      --disable-R-framework
```

Comment:
Hmm, I suppose that flag enables a build variant of `R` so that the package can be used as a framework, i.e. it probably enables some Darwin-specific magic with regard to shared libraries. I'm not sure ... but, well, it probably doesn't matter to us in Nix.